lib/Robo/Plugin/Commands/BuildCommands.php

```diff
@@ -57,7 +57,7 @@ class BuildCommands extends \Robo\Tasks
      * color_scheme - set which color scheme you wish to build
      * @throws \RuntimeException
      */
-    public function buildSuiteP(array $opts = ['color_scheme' => ''])
+    public function buildSuiteP(array $opts = ['color-scheme' => ''])
```

Comment:
You need to change it for the rest of the file as well :P

Reply:
It's too Friday afternoon 🤦🏼‍♂